cpp/pipelines/transfer_880_translations.cc

```diff
@@ -0,0 +1,102 @@
+/** \file    transfer_880_translations
+ *  \brief   Transfer translations found in 880 to match with our ordinary 750 translations
+ *  \author  Johannes Riedl
+ */
+
+
+/*
+    Copyright (C) 2020 Library of the University of Tübingen
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as
+    published by the Free Software Foundation, either version 3 of the
+    License, or (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <iostream>
+#include "MARC.h"
+#include "util.h"
+
+
+namespace {
+
+const std::string UNDEFINED_LANGUAGE("Undefined");
+
```

Comment:
Ditto.  We always have exactly 2 blank lines between file-scope entities, declarations functions etc.  There are *no* special rules!